ragen/env/lean/env.py

```
@@ -0,0 +1,647 @@
+            logger.error(message)
+            raise AssertionError(message)
+
+        from datasets import load_dataset
+
+        try:
+            hf_dataset = load_dataset(
+                dataset_name,
+                split=dataset_split,
+                streaming=False,
+            )
+        except Exception as exc:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)